src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,6 +14,9 @@
     private Phone phone;
     private Email email;
     private Address address;
+    
+    private int sequenceNumber;
+    private static int nextSequenceNumber;
```

Comment:
don't you think semantically it makes more sense to start next sequence number from 1?